pkg/utils/authenticationUtils.go

```diff
@@ -67,3 +69,37 @@ func readJWTFromFS(tokenFilePath string) (string, error) {
 	}
 	return string(token), nil
 }
+
+// RegionFromImage parses region from image url
+func RegionFromImage(image string) (string, error) {
+	registry, err := RegistryFromImage(image)
+	if err != nil {
+		return "", err
+	}
+
+	region := RegionFromRegistry(registry)
+
+	return region, nil
+}
+
+// RegionFromRegistry parses AWS region ID from registry url
+func RegionFromRegistry(registry string) string {
+	a := strings.Split(registry, ".")
+	if len(a) >= 6 {
+		return a[3]
+	}
+	return ""
+}
+
+// RegistryFromImage parses registry host from image url
```

Comment:
@akashsinghal @binbin-li does ORAS has a method to ParseReference o return the host from the image reference?

Reply:
Yes there is one here: https://github.com/oras-project/oras-go/blob/e8225cb1e125bd4c13d6b586ae6d862050c3fae2/registry/reference.go#L114
@jimmyraywv you could use this instead